### ISSUE TITLE
Need to home before can move.

### ISSUE THREAD

**Issue by Jeremyrab** [Silverquark/i3PlusPlus#42]
Whenever I want to move on any of the axis they have to go home first. This happens every time I turn on the printer. Is there a reason for this because the original firmware didn't have to do this.

**Comment by Silverquark** [comment]
Yes, there is a reason for it. It homes to know it's current Position. After that it Wont go over Max/min positions to prevent accidently ramming into the printer frame. I originally planned to make this Feature optional but havent yet
